AlpsSDK/AlpsManager/AlpsManager.swift

```diff
@@ -47,6 +47,7 @@ public class AlpsManager: MatchMonitorDelegate, ContextManagerDelegate {
         if let baseURL = baseURL {
             self.baseURL = baseURL
         }
+        remoteNotificationManager = RemoteNotificationManager(delegate: matchMonitor)
```

Comment:
At first, I wanted to use `AlpsManager` to be the delegate. 

But this is creating one more indirection so I went straight for `MatchMonitor`, what do you think ?

Reply:
Looks reasonable.